tools.yml

```diff
@@ -492,82 +505,15 @@ tools:
     mem: 40
   toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_tblastx_wrapper/.*:
     cores: 16
+    mem: 64
   toolshed.g2.bx.psu.edu/repos/devteam/picard/.*:
```

Comment:
I don't find any picard tool in your gist, deleting them would mean a mem increase for the `12G` picard tools to `~15.2G` and I am not sure if we need the `JAVA_OPTIONS` or if they are set in the wrapper now

Reply:
This should make all picard tools 12G by matching just the owner/repo combo of `toolshed.g2.bx.psu.edu/repos/devteam/picard/.*`. This entry already existed with `mem: 12` whereas all the specific tool entries I removed also set `mem: 12`, so they were just redundant.

The tools definitely need `-Xmx` set, although some local TPV configs may already do this (I set it in `defaults` for all my tools, regardless of whether they are java, because it's harmless if they're not).

Reply:
> The tools definitely need -Xmx set, although some local TPV configs may already do this (I set it in defaults for all my tools, regardless of whether they are java, because it's harmless if they're not).

This is good to know, thank you :)